src/frontend/components/Felleskomponenter/UtbetalingerModal/UtbetalingerModal.tsx

```diff
@@ -114,41 +119,98 @@ export const UtbetalingerModal: React.FC<UtbetalingerModalProps> = ({
+                                            UtbetalingerSpørsmålId.utbetalingTilDato
+                                        )}
+                                        values={{ ...(barn && { barn: barn.navn }) }}
+                                    />
+                                }
+                                tidligsteValgbareMåned={minTilDatoForUtbetalingEllerArbeidsperiode(
+                                    periodenErAvsluttet,
+                                    utbetalingFraDato.verdi
+                                )}
+                                senesteValgbareMåned={
+                                    periodenErAvsluttet ? sisteDagDenneMåneden() : undefined
+                                }
+                                felt={utbetalingTilDato}
+                                visFeilmeldinger={skjema.visFeilmeldinger}
+                                dagIMåneden={DagIMåneden.SISTE_DAG}
+                                kanIkkeVæreFremtid={periodenErAvsluttet}
+                                kanIkkeVæreFortid={!periodenErAvsluttet}
+                                disabled={utbetalingTilDatoUkjent.verdi === ESvar.JA}
+                            />
+                            <SkjemaCheckbox
+                                labelSpråkTekstId={hentSpørsmålTekstId(
+                                    UtbetalingerSpørsmålId.utbetalingTilDatoVetIkke
+                                )}
+                                felt={utbetalingTilDatoUkjent}
+                            />
+                        </>
+                    ) : (
+                        <>
+                            <Datovelger
+                                skjema={skjema}
+                                felt={utbetalingTilDato}
+                                label={
+                                    <SpråkTekst
+                                        id={hentSpørsmålTekstId(
+                                            UtbetalingerSpørsmålId.utbetalingTilDato
+                                        )}
+                                    />
+                                }
+                                avgrensMaxDato={periodenErAvsluttet ? dagensDato() : undefined}
+                                avgrensMinDato={minTilDatoForUtbetalingEllerArbeidsperiode(
+                                    periodenErAvsluttet,
+                                    utbetalingFraDato.verdi
+                                )}
+                                disabled={utbetalingTilDatoUkjent.verdi === ESvar.JA}
+                            />
+                            <SkjemaCheckbox
+                                labelSpråkTekstId={hentSpørsmålTekstId(
+                                    UtbetalingerSpørsmålId.utbetalingTilDatoVetIkke
+                                )}
+                                felt={utbetalingTilDatoUkjent}
+                            />
+                        </>
+                    )}
```

Comment:
Samme som over. Ny `MånedÅrVelger` for fom og tom i `if` og gammel `Datovelger` i `else`. `SkjemaCheckbox` utenfor.

Reply:
Enig, fikset det nå 😃